deploy/kube-config/influxdb/heapster-controller.json

```diff
@@ -26,7 +26,7 @@
 	    "spec": {
 		"containers": [
 		    {
-			"image": "kubernetes/heapster:v0.18.1",
+			"image": "thuctnguyen/heapster:v0.18.2-rc1",
```

Comment:
I built this image from HEAD. I think it'd be a good idea to have kubernetes/heapster:latest, which gets built automatically from HEAD.